README.md

```diff
@@ -1,3 +1,164 @@
 # cogitate_tool
 
-Tool to analyze contributions when working in a team.
+## Development Doc
+
+After pulling the repo, use `pipenv shell` in `cogitate_tool/` to enter the virtual
+environment. Use `exit` to exit. Under the virtual environment, use
+`pipenv install <package_name> --dev` to install new packages.
+
+### 1. File Structure
+
+```bash
+.
+├── cogitate_tool.py ------------- *main function*
+├── driller.py ------------------- *PyDriller*
+├── Pipfile ---------------------- *Package file*
+├── Pipfile.lock ----------------- *Package version lock file*
+├── pyinquirer ------------------- *CLI demo*
+│   ├── hello.py ----------------- *Demo Hello World*
+│   ├── img ---------------------- *Images for markdown*
+│   │   ├── interactive.png
+│   │   └── result.png
+│   └── intactv.py --------------- *Examples for choices question*
+└── README.md
+```
```

Comment:
This structure looks good! Talk to the architecture team/architect about this so that it will be adopted in the master. But some changes still need to be made. For example, the `img` should be outside of `pyinquirer` i think. All the actual code modules should be inside of a directory, maybe called `cogitate`. You also want to have a directory for all the test cases like `tests`. It will look a lot like what you guys have in the practical assignment